src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -61,6 +61,7 @@ function getSentryIntegrations(hasReplays: boolean = false) {
         Router.createRoutes(routes()),
         Router.match as any
       ),
+      idleTimeout: 2000,
```

Comment:
This will be overridden by https://github.com/getsentry/sentry/pull/20738

Reply:
Just as a heads up Alberto, I think that could increase transaction times, as now whatever the users does within 5 seconds will extend the transaction.